bump_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
```

Comment:
This line should point to the correct file for your project.

Reply:
@jsf9k I made that fix here - https://github.com/cisagov/domain-manager-cli/pull/75/commits/6ddaf1b4265df4d714ba0e45d8d517b0eb665eec